search_that_hash/cracker/sth_mod/sth.py

```diff
@@ -0,0 +1,37 @@
+import json
+import requests
+
+
+def crack(config):
+    to_del = []
+    url = "https://av5b81zg3k.execute-api.us-east-2.amazonaws.com/prod/lookup"
+    payload = json.dumps({"Hash": list(config["hashes"].keys())})
+    headers = {"Content-Type": "application/json"}
+    try:
+        response = requests.request(
+            "GET", url, headers=headers, data=payload, timeout=config["timeout"]
+        )
+    except:
+        return False, config

```

Comment:
Use specific exceptions such as `except TypeError` instead of this, it makes it extremely hard to understand what the function Is doing. If you cant find a specific error then you don't need a try except as it'll never error.